mlflow/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM --platform=linux/amd64 python:3.8.10 AS build
```

Comment:
I think it will be better to use the official MLflow Docker image directly

https://mlflow.org/docs/latest/docker.html